planning/autoware_obstacle_stop_planner/schema/adaptive_cruise_control.schema.json

```diff
@@ -0,0 +1,209 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameter for adaptive cruise control",
+  "type": "object",
+  "definitions": {
+    "adaptive_cruise_control": {
+      "type": "object",
+      "properties": {
+        "use_object_to_estimate_vel": {
+          "type": "boolean",
+          "description": "use tracking objects for estimating object velocity or not",
+          "default": "true"
+        },
```

Comment:
CI is complaining about adaptive_cruise_control.schema.json
It's probably because the param file has [adaptive_cruise_control as a top level namespace](https://github.com/autowarefoundation/autoware.universe/blob/271274c2ef4fb7c89de9b283499c73f1b93373b9/planning/autoware_obstacle_stop_planner/config/adaptive_cruise_control.param.yaml#L3).

![image](https://github.com/autowarefoundation/autoware.universe/assets/43976834/0839886f-e550-412f-b06b-f315a54eada1)

```suggestion
    "adaptive_cruise_control": {
      "type": "object",
      "properties": {
        "adaptive_cruise_control": {
          "type": "object",
          "properties": {
          "use_object_to_estimate_vel": {
            "type": "boolean",
            "description": "use tracking objects for estimating object velocity or not",
            "default": "true"
          },
```